frontend_vue/package.json

```diff
@@ -53,6 +53,7 @@
     "sass": "^1.71.1",
     "tailwindcss": "^3.4.1",
     "typescript": "~5.3.0",
+    "unplugin-vue-components": "^0.26.0",
```

Comment:
whats the benefit of this besides not having to import things?

Reply:
We don't have to export them manually: to make typescript work correctly with the Base components ( meaning: perform a type checking on props for instance ) we had to export them manually, by including them in the globalcomponents.d.ts file